src/main/java/biz/paluch/logging/gelf/intern/sender/GelfKafkaSenderProvider.java

```diff
@@ -0,0 +1,78 @@
+package biz.paluch.logging.gelf.intern.sender;
+
+import biz.paluch.logging.gelf.PropertyProvider;
+import biz.paluch.logging.gelf.intern.GelfSender;
+import biz.paluch.logging.gelf.intern.GelfSenderConfiguration;
+import biz.paluch.logging.gelf.intern.GelfSenderProvider;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.common.serialization.ByteArraySerializer;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+/**
+ * @author Rifat Döver
+ * @since 1.13
+ *
+ */
+public class GelfKafkaSenderProvider implements GelfSenderProvider {
```

Comment:
I'm not sure whether adding another configuration system is a good idea. Why can't we stick to URL-like properties that are transported through the `host` field?

Reply:
Kafka Producer has very complex configuration options, especially about security.
Please see(https://docs.confluent.io/current/installation/configuration/producer-configs.html)
For Redis implementation, there is RedisGelfSenderProvider also.
Or am I understanding wrong what you are trying to mean?

Reply:
Having `KafkaGelfSenderProvider` is perfectly fine to configure and instantiate the sender.